Makefile

```diff
@@ -355,8 +359,10 @@ check: protoc ## Dockerized version of make test
 	$(DOCKER_GO) "make test"
 
 .PHONY: container-build
-container-build: ## Build containers
```

Comment:
NIT: It might be nice to keep this target for building the containers locally.

Reply:
I did keep it, git view might be a bit confusing since it's showing it was removed (here in red) and later on added (in green)